docs/docs/code/go/README.md

```diff
@@ -1,6 +1,452 @@
 # Go
 
+**Anything you can do in Go can be done in a Pipedream Workflow**. This includes using any of the [Go packages available](https://pkg.go.dev/) in your Go powered workflows. 
 
-## HTTP requests
+Pipedream supports [Go v{{$site.themeConfig.GO_LANG_VERSION}}](https://go.dev) in workflows.
 
-https://pkg.go.dev/net/http
+::: warning
+Go steps are available in a limited alpha release.
+
+You can still run arbitrary Go code, including [sharing data between steps](/code/go/#sharing-data-between-steps) as well as [accessing environment variables](/code/go/#using-environment-variables).
+
+However, features available in [Node.js steps](/code/nodejs) like `$.respond`, `$.end`, and `$.auth` are not yet available in bash. If you have any questions please [contact support](https://pipedream.com/support).
+:::
+
+
+## Adding a Go code step
+
+1. Click the + icon to add a new step
+2. Click "Custom Code"
+3. In the new step, select the `golang` language runtime in language dropdown
+
+## Logging and debugging
+
+You can use `fmt.Println` at any time in a Go code step to log information as the script is running.
+
+The output for the `fmt.Println` **logs** will appear in the `Results` section just beneath the code editor.
+
+:::tip
+Don't forget to import the `fmt` package in order to run `fmt.Println`.
+
+```go{3}
+  package main
+
+  import "fmt"
+
+  func main() {
+    fmt.Println('Hello World!')
```

Comment:
You'll need to use double-quotes to define strings. Single-quotes are for "rune literals". This code fails with the error:

<img width="1001" alt="Screen Shot 2022-01-30 at 10 16 14 PM" src="https://user-images.githubusercontent.com/242668/151747471-58f05d40-7311-4aed-84d8-3efcde22dfc2.png">